src/api/entities/Instruction/index.ts

```diff
@@ -966,7 +966,10 @@ export class Instruction extends Entity<UniqueIdentifiers, string> {
         if (mediatorAffirmation) {
           status = middlewareAffirmStatusToAffirmationStatus(mediatorAffirmation.status);
           if (mediatorAffirmation.expiry) {
-            return { status, expiry: mediatorAffirmation.expiry };
+            return {
+              status,
+              expiry: mediatorAffirmation.expiry ? new Date(mediatorAffirmation.expiry) : undefined,
```

Comment:
```suggestion
              expiry: new Date(mediatorAffirmation.expiry),
```
There's a check on line 968 already. 

You'll never be able to get branch coverage here.